CMakeLists.txt

```diff
@@ -12,25 +12,8 @@ find_package(Boost REQUIRED system filesystem)
 find_package(ASSIMP QUIET)
 if (NOT ASSIMP_FOUND)
   find_package(PkgConfig REQUIRED)
-  pkg_check_modules(ASSIMP assimp)
-endif()
-if (ASSIMP_FOUND)
```

Comment:
fyi Indigo also [targets Ubuntu 13.10](http://www.ros.org/reps/rep-0003.html#jade-turtle-may-2015-may-2017), but I just checked and it looks like that also provides [assimp 3.0](https://launchpad.net/ubuntu/saucy/+source/assimp) so I guess this is good. would feel better about all these changes if they were for jade/kinetic